tests/integration/integ-suite-kind.sh

```diff
@@ -30,6 +30,9 @@ export DEFAULT_CLUSTER_YAML="${SCRIPTPATH}/config/default.yaml"
 export IP_FAMILY="${IP_FAMILY:-ipv4}"
 export ARTIFACTS="${ARTIFACTS:-$(mktemp -d)}"
 
+# Set variable for cluster kind name
+export CLUSTER_NAME="${CLUSTER_NAME:-istio-operator}"
```

Comment:
We could rename this variable to be more explicit like `KIND_CLUSTER_NAME`.
Also, the default value could be something like `operator-integration-tests`...

I mean, these are attempts to avoid future clashes.

Reply:
+1

Reply:
Done

Reply:
Can you also rename to `operator-integration-tests`?

Reply:
Done